applications.yaml

```diff
@@ -678,6 +678,13 @@
     id: _WwB
     comment: Targets a hidden window spawned by Microsoft Office applications
     matching_strategy: Legacy
+- name: Microsoft SQL Server Management Studio
+  identifier:
+    kind: Exe
+    id: Ssms.exe
+    matching_strategy: Legacy
```

Comment:
This can be `Equals` since we are matching an an `Exe` name

Reply:
Cool good to know thanks